bin/test-pytest

```diff
@@ -7,4 +7,4 @@ cd "$(dirname "$0")/.."
 echo 'Running pytest...'
 
 coverage erase
-PYTHONWARNINGS='error:::betty[.*]' pytest --cov --cov-append --cov-config=.coveragerc --no-cov-on-fail "$@"
+PYTHONWARNINGS='error:::betty' pytest --cov --cov-append --cov-config=.coveragerc --no-cov-on-fail "$@"
```

Comment:
Remove `PYTHONWARNINGS` as we now handle conversion to errors using a test fixture.